verilog/CST/module.cc

```diff
@@ -97,18 +110,23 @@ const TokenInfo* GetModuleEndLabel(const verible::Symbol& s) {
   return &module_name.get();
 }
 
-const verible::SyntaxTreeNode& GetModuleItemList(
+const verible::SyntaxTreeNode* GetModuleItemList(
     const verible::Symbol& module_declaration) {
-  return verible::GetSubtreeAsNode(module_declaration,
-                                   NodeEnum::kModuleDeclaration, 1,
-                                   NodeEnum::kModuleItemList);
+  const SyntaxTreeNode& module_node =
+      verible::SymbolCastToNode(module_declaration);
+  if (!module_node.MatchesTagAnyOf({NodeEnum::kModuleDeclaration,
+                                    NodeEnum::kInterfaceDeclaration,
+                                    NodeEnum::kProgramDeclaration})) {
```

Comment:
Since this logic appears more than one place, factor out a function for this somewhere.